qcli/bsp2svg/cli.py

```diff
@@ -71,7 +79,7 @@ def main():
     # Validate or create out file
     if args.dest == os.getcwd():
         svg_path = os.path.dirname(args.file)
-        svg_name = f'{os.path.basename(args.file).split(".")[0]}.svg'
+        svg_name = f'{os.path.basename(args.file).split(".")[0]}_{args.projection_axis}.svg'
```

Comment:
I like this!

Reply:
thanks!
I just had another idea. the ability to specify a negative axis, in order to flip the projection. this makes more sense for the x and y axis. what do you think? should I add this? how should I put the choices? something like `choices=['x', 'y', 'z', 'nx', 'ny', 'nz']`?

Reply:
Maybe hold off on this for the moment? You can also accomplish this by manually flipping the svg drawing, correct?

Reply:
almost. the outer edge could look same when flipping the svg, but the content inside could be different. but yes, we can hold this off